static/js/base/ga.js

```diff
@@ -0,0 +1,60 @@
+/* global dataLayer */
+const origin = window.location.href;
+const categoryPrefix = "snapcraft.io-";
+
+const events = {
```

Comment:
We should probably kill the current events on details pages since they are not sent to the combo tracker therefore don't allow for viewing cross site funnels. I'll adjust the existing reports depending on them.

So, my suggestion would be to track clicks on: 
`p-view-store-button` (action: `from:{origin} to:{snap://package-name}`)
`p-snap-install-buttons__versions` (action: `from:{origin} to:{the content of "data-control"}`)
`p-snap-install-buttons__install` (action: `from:{origin} to:{the content of "data-control"}`)

Thoughts?

Reply:
Makes sense, should they be `cta-1` categories I guess?

Reply:
Looking at it from a user perspective, cta-0 (primary call to action) for "p-snap-install-buttons__install" and cta-1 for the others.